.infra/index.ts

```diff
@@ -63,8 +63,8 @@ if (isAdhocEnv) {
 const redis = new Redis(`${name}-redis`, {
   isAdhocEnv,
   name: `${name}-redis`,
-  tier: 'STANDARD_HA',
-  memorySizeGb: 10,
+  tier: 'BASIC',
```

Comment:
It may fail to downgrade so let's keep an eye

Reply:
Yeah, it might. It could probably go through via click-ops if pulumi fails.
I'd also like to do it on saturday/sunday morning to reduce any chances of it evicting all keys.